### DISCUSSION TITLE
Quote Problem

### DISCUSSION THREAD

**Discussion by ForbAvalon** [Truinto/DarkCodex#109] [Q&A] [answered]
When I clone this repository with VS , I find that this project quote some files with _publicized
Where can I find these file ? Unity2019.4.26f1 ?

**Comment by Truinto** [comment, marked as answer]
Weird my answer didn't make it the first time.

These are publicized assemblies. Meaning assemblies manipulated to have everything public. For copyright reasons I cannot share these files. But you can create them yourself. You need this
https://github.com/CabbageCrow/AssemblyPublicizer/tree/v1.0.0
or this
https://github.com/WittleWolfie/OwlcatModdingWiki/wiki/Publicize-Assemblies

**Reply by ForbAvalon** [reply]
Thanks you

**Comment by Truinto** [comment]
https://github.com/CabbageCrow/AssemblyPublicizer

https://github.com/WittleWolfie/OwlcatModdingWiki/wiki/Modding-Resources

You need to run the assemblies through this program. It pretends everything is public, which isn't checked during execution.
